### DISCUSSION TITLE
AsusFX504 - 0.8.6 2nd RELEASE

### DISCUSSION THREAD

**Discussion by RobyRew** [RobyRew/ASUS-FX504GE-Hackintosh_OpenCore#19] [General]
### News / Fixes:
- Fixed HDMI not working at all because someone at Whatevergreen changed it functionality `agdpmod | Data | 76697439363936` (credits: https://www.reddit.com/r/hackintosh/comments/10ywrx6/comment/j804eqb/)

### Be aware:
- Right now this EFI is using [Airportitlwm.kext v2.2.0-Alpha(ee56708)](https://github.com/OpenIntelWireless/itlwm/releases/tag/v2.2.0-alpha) because its using Alpha software it might have some issues unresolved, but nothing "Stable" is provided for Ventura yet.

### Known Issues:
- HDMI may not work great on certain displays especially if connected to a High Res Display.
- Audio sometimes or in some situations is not great.

_Pull Requests are welcome! :)_

<hr /><em>This discussion was created from the release <a href='https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_OpenCore/releases/tag/0.8.6-1'>AsusFX504 - 0.8.6 2nd RELEASE</a>.</em>

**Comment by Dv1101** [comment]
Sonoma booting?

**Comment by RobyRew** [comment]
not tested sorry

**Comment by Dv1101** [comment]
We r having intel bt chip, Why it is Broadcom controller, Im on Ventura and BT is not working. Here is EFI SS
![Screenshot 2023-07-29 at 1 09 43 PM](https://github.com/RobyRew/ASUS-FX504GE-Hackintosh_OpenCore/assets/86505071/c26aac30-5ddf-479c-9898-ac28d451a25d)

**Reply by RobyRew** [reply]
Its the only way that we can have Bluetooth somehow working, otherwise there's anything to bring Bluetooth support for intel 
Here's Why: https://openintelwireless.pages.dev/IntelBluetoothFirmware/FAQ.html
And how to be installed: https://openintelwireless.github.io/IntelBluetoothFirmware/FAQ.html#what-additional-steps-should-i-do-to-make-bluetooth-work-on-macos-monterey-and-newer

PS: I hope I'll update this for Sonoma soon

**Comment by Dv1101** [comment]
@RobyRew I booted sonoma from Ventura efi, wifi - bt - audio is dead, display brightness can't able to change, touchpad working. For wifi and bt have to switch newer kext itlwm, will try. For audio I don't know, in system info it shows as driver not installed

**Reply by RobyRew** [reply]
I will take a look these days if I have some time

**Comment by Dv1101** [comment]
New stable kext are working flawlessly for Ventura, Wifi is working perfectly fine with best speed, BT also working now :)

**Comment by RobyRew** [comment]
Hey @Dv1101 the EFI is now updated with OpenCore 0.9.7 for macOS Sonoma, tested, everything is working great for mee, except HDMI, i'll have to digg more about it
